web/utils.py

```
@@ -942,3 +942,28 @@ def geocode_address(address):
+        "dropped": "danger",
+        "spamreport": "danger",
+        "deferred": "warning",
+        "processed": "info",
+    }
+    status_class = status_classes.get(profile.last_email_event, "info")
+    color = {"success": "green", "danger": "red", "warning": "orange", "info": "blue"}.get(status_class, "gray")
+    last_event_time = getattr(profile, "last_email_event_time", None)
+    if last_event_time:
+        time_str = last_event_time.strftime("%Y-%m-%d %H:%M")
+    else:
+        time_str = ""
```

Ruff (0.8.2)

Use ternary operator `time_str = last_event_time.strftime("%Y-%m-%d %H:%M") if last_event_time else ""` instead of `if`-`else`-block

Replace `if`-`else`-block with `time_str = last_event_time.strftime("%Y-%m-%d %H:%M") if last_event_time else ""`

(SIM108)